pkg/kv/kvserver/store_snapshot.go

```diff
@@ -338,6 +368,22 @@ func (kvSS *kvBatchSnapshotStrategy) Receive(
 				raftAppliedIndex:  header.State.RaftAppliedIndex,
 			}
 
+			totalTimeStopwatch.Stop()
+			totalTime := totalTimeStopwatch.Elapsed().Seconds()
+			msstwDuration := msstwStopwatch.Elapsed().Seconds()
+			batchDuration := batchStopwatch.Elapsed().Seconds()
+			recvDuration := recvStopwatch.Elapsed().Seconds()
+
+			log.Infof(
```

Comment:
Will this be spammy for every snapshot? Instead of log.Infof, stashing these in the traces alone seem good enough.